README.md

```diff
@@ -28,12 +30,22 @@ These instructions are intended for those wishing to examine the Synthea source
 Synthea<sup>TM</sup> requires Java 1.8 or above.
 
 To clone the Synthea<sup>TM</sup> repo, then build and run the test suite:
-```
+```4
```

Comment:
Looks like a random "4" typo, otherwise these are acceptable.

Reply:
4 removed.  It must have been the cat typing.